src/pages/Settings/Settings.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react';
+import { useTheme } from '../../hooks/useDarkMode';
+import { Moon, Sun } from 'lucide-react';
+
+const Settings = () => {
+  const { isDarkMode, toggleDarkMode } = useTheme();
+  const [notifications, setNotifications] = useState(false);
+  const [studyReminder, setStudyReminder] = useState('daily');
+
+  return (
+    <div className="p-4">
+      <h1 className="text-2xl font-bold mb-6">Settings</h1>
+
+      <div className="space-y-4">
+        {/* Dark Mode s ikonom */}
+        <div className="bg-white dark:bg-gray-800 p-4 rounded-xl shadow-sm">
```

Comment:
i see too many repeating yourself. maybe creating one component would help us have more readable and more simple code.